share/picongpu/benchmarks/Thermal/README.rst

```diff
@@ -0,0 +1,11 @@
+Thermal electron benchmark
+==========================
+
+.. sectionauthor:: Rene Widera <r.widera (at) hzdr.de>
+
+This setup simulates a homogeneous electron plasma.
+There are 25 macro-particles per cell which sample a homogeneous distribution of relativistic electrons
+at a density of ρWP = −10^20 e/m3.
+Their initial momenta are normally distributed along each spatial dimension with a variance of 17.5 m^2c^2,
```

Comment:
I think this is wrong? Momenta follow Maxwell-Boltzmann, so not independent on each axis and the variance is for absolute value of momentum